### PR TITLE
2.2.17: making keep/unkeep/private work in mitro

### PR DESCRIPTION
**Pull request by 2is10**
Also improving the deep link redirect page:
- clarifying in the message that we're talking about the user's current browser
- giving the KiFi extension up to 100ms after DOM ready to hide the message before we show it

### CODE REVIEW

**Comment by 2is10** [comment]
In case you were wondering, _in mitro_ is a latin phrase meaning, roughly, "in metro".

File: packrat/main.js [hunk 1]
```diff
@@ -140,11 +145,11 @@ api.port.on({
     }
     return true;
   },
-  get_slider_updates: function(data, respond, tab) {
+  get_slider_updates: function(_, respond, tab) {
```
Comment (eishay) [inline packrat/main.js:148]: what does underscore means? Something like the 'i don't care' in scala?

Comment (2is10) [inline packrat/main.js:148]: That's how I'm using it here, yes. I haven't seen the pattern in JavaScript before, but it works, so I thought I'd try it out.

Comment (eishay) [inline packrat/main.js:148]: cool :-)

**Pull request merged**